modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerScaleUpTest.java

```diff
@@ -307,312 +301,6 @@ void testDropZoneDoNotPropagateDataNodesAfterScaleDown() throws Exception {
         assertZoneScaleDownChangeTriggerKey(null, ZONE_1_ID, keyValueStorage);
     }
 
-    @Test
-    void testTwoScaleUpTimersSecondTimerRunFirst() throws Exception {
```

Comment:
Is this a temporary solution? If yes, why did we delete this tests, not mute?

Reply:
This is a permanent solution. My algorithm needs a guarantee that tasks of each zone will be executed in the order they were added to the executor. These tests check that multiple tasks for a zone are running at the same time, so now such tests will not pass successfully. Even if I make a striped executor, it won't change this.